OpenRA.Mods.Common/Lint/CheckFluentReferences.cs

```diff
@@ -80,29 +78,24 @@ void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData
 			foreach (var context in usedKeys.EmptyKeyContexts)
 				emitWarning($"Empty key in mod translation files required by {context}");
 
-			var allModTranslations = modData.Manifest.Translations.ToArray();
-			foreach (var language in GetModLanguages(allModTranslations))
+			var modMessages = modData.Manifest.FluentMessages.ToArray();
+			CheckModWidgets(modData, usedKeys, testedFields);
+
+			// With the fully populated keys, check keys and variables are not missing and not unused across all language files.
+			var keyWithAttrs = CheckKeys(
+				modMessages, modData.DefaultFileSystem.Open, usedKeys,
+				file =>
+					!modData.Manifest.AllowUnusedFluentMessagesInExternalPackages ||
+					!modData.DefaultFileSystem.IsExternalFile(file),
+				emitError, emitWarning);
+
+			foreach (var group in usedKeys.KeysWithContext)
 			{
-				Console.WriteLine($"Testing language: {language}");
```

Comment:
We should keep a console print to be consistent with other major lint test

Reply:
Ok, done.